packages/dds/tree/src/shared-tree-core/sharedTreeCore.ts

```diff
@@ -434,28 +444,27 @@ export interface Summarizable {
 	 */
 	readonly key: string;
 
-	/**
-	 * {@inheritDoc @fluidframework/datastore-definitions#(IChannel:interface).getAttachSummary}
-	 * @param stringify - Serializes the contents of the component (including {@link (IFluidHandle:interface)}s) for storage.
-	 */
-	getAttachSummary(
-		stringify: SummaryElementStringifier,
-		fullTree?: boolean,
-		trackState?: boolean,
-		telemetryContext?: ITelemetryContext,
-		incrementalSummaryContext?: IExperimentalIncrementalSummaryContext,
-	): ISummaryTreeWithStats;
-
 	/**
 	 * {@inheritDoc @fluidframework/datastore-definitions#(IChannel:interface).summarize}
 	 * @param stringify - Serializes the contents of the component (including {@link (IFluidHandle:interface)}s) for storage.
+	 * @param fullTree - flag indicating whether the attempt should generate a full
```

Comment:
Nit: can we call out the default value explicitly here? Also, I'm pushing for us to use full sentence casing consistently when possible.
```suggestion
	 * @param fullTree - A flag indicating whether the attempt should generate a full
```